sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/base64url.hpp

```diff
@@ -52,7 +52,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _internal
           base64url.append("=");
           break;
         default:
-          throw new std::invalid_argument("Unexpected Base64URL encoding in the HTTP response.");
+          throw std::invalid_argument("Unexpected Base64URL encoding in the HTTP response.");
```

Comment:
I opened an https://github.com/Azure/azure-sdk-for-cpp/issues/317 about this kind of leaks before. But they weren't all fixed. Thanks @antkmsft

Reply:
@Jinming-Hu they were fixed in core when you brought it up, this got introduced recently in new code within keyvault.

Reply:
https://media1.tenor.com/images/b19a7e5ca8b0bb2fd56f6c7cf3278131/tenor.gif?itemid=16311175